docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+
+services:
+  envelop:
+    build: .
+    ports:
+      - "3000:3000"
+    env_file:
+      - .env
```

Comment:
but we have `.env.example` right? it will work? maybe we can add `.env.development`